cg/apps/demultiplex/sample_sheet/api.py

```diff
@@ -33,6 +34,11 @@
 from cg.models.flow_cell.flow_cell import FlowCellDirectoryData
 from cg.utils.files import get_directories_in_path, link_or_overwrite_file
 
+FLOW_CELL_SAMPLE_TO_BCL_CONVERTER: dict[Type[FlowCellSample], str] = {
+    FlowCellSampleBcl2Fastq: BclConverter.BCL2FASTQ,
```

Comment:
why do we need this mapper? is it not always a "bcl2fastq" to "bcl convert" conversion?

Reply:
The validation command can still manage both sample sheet formats. If no format is specified, it infers it from the content of the sheet. The function that we currently have returns (unfortunately) the sample model (FlowCellSampleBcl2Fastq or FlowCellSampleBCLConvert) and we need the converter as a `str`. This mapper is what I thought to convert the sample model to bcl converter.